test/support/stripe_mock.ex

```diff
@@ -41,26 +41,35 @@ defmodule Stripe.StripeMock do
 
   @impl true
   def handle_info({:stdout, os_pid, msg}, %{os_pid: os_pid} = state) do
-    Logger.debug("[stripe-mock:out] #{String.trim msg}")
+    Logger.debug("[stripe-mock:out] #{String.trim(msg)}")
     {:noreply, state}
   end
 
   @impl true
   def handle_info({:stderr, os_pid, msg}, %{os_pid: os_pid} = state) do
-    Logger.debug("[stripe-mock] #{String.trim msg}")
+    Logger.debug("[stripe-mock] #{String.trim(msg)}")
     {:noreply, state}
   end
 
   @impl true
-  def handle_info({:DOWN, os_pid, :process, _ex_pid, _reason}, %{os_pid: os_pid, opts: opts, restarting: {true, from}} = state) do
+  def handle_info(
+        {:DOWN, os_pid, :process, _ex_pid, _reason},
+        %{os_pid: os_pid, opts: opts, restarting: {true, from}} = state
```

Comment:
They recommend doing this match as the first line of the function since it's immaterial to the function being invoked.